templates/index.html

```diff
@@ -40,7 +40,9 @@ <h2><a href="{{ SITEURL }}/{{ article.url }}{% if not DISABLE_URL_HASH %}#{{ art
       {% endif %}
       {{ article.summary }}
       <br>
-      <a class="btn" href="{{ SITEURL }}/{{ article.url }}{% if not DISABLE_URL_HASH %}#{{ article.slug }}{% endif %}">{{ _('Continue reading') }}</a>
+      {% if article.content|length > article.summary|length %}
```

Comment:
summary and content are two different properties, what if my summary has 100 chars and my content has 50 but the text of the summary is not the same?

My guess `{% if article.content != article.summary %}` is a better way to check for this case.